build.gradle

```diff
@@ -27,6 +28,9 @@ dependencies {
 
     // For the @CalledMethodsPredicate evaluation
     implementation 'org.springframework:spring-expression:5.1.7.RELEASE'
+    
+    // For returns receiver checking
+    implementation 'org.checkerframework:returnsrcvr-checker:0.1-SNAPSHOT'
```

Comment:
Maybe we can use https://jitpack.io/ to pull in this dependence without having to upload to Maven Central?  This would only be temporary.